requirements-freeze.txt

```diff
@@ -40,7 +40,6 @@ PasteDeploy==1.5.0
 PasteScript==1.7.5
 pbr==0.8.2
 pika==0.9.8
-pip>=19.2
```

Comment:
@kmanaseryan I'm not sure about this line. Could you keep it and only include the `ckanext-dcat_usmetadata` related change?

Reply:
@anuveyatsu I didn't make any updates on this file directly. Just run the command ` make clean build update-dependencies requirements`. I also noticed this, should I add this bring this line back?